### PR TITLE
Tape Wizard, the distant cousin of the Paper Wizard

### PR DESCRIPTION
**Pull request by Draggeru**
## About The Pull Request

Adds a new wizard costume and non magical variant, made exclusively from tape.

qTwAAAAASUVORK5CYII

## Why It's Good For The Game

New unique nice looking thing = good 
Opens the door for more uses of an already niche mechanic
ducktape
Roleplay.

## Changelog
:cl:
add: Adds the tape wizard costume, both a real and fake variant
add: Adds the costume behind the autodrobe contraband wire, and the real variant in the wiz den
:cl:

### CODE REVIEW

**Comment by Draggeru** [comment]
I will make the kit have super sticky tape instead if the maintainers approve of people having such power near roundstart

File: code/game/objects/effects/spawners/costume.dm [hunk 1]
```diff
@@ -167,6 +167,14 @@
 		/obj/item/clothing/suit/wizrobe/marisa/fake,
 	)
 
+/obj/effect/spawner/costume/tapewizard
```
[the remark this answers is not in the document]
Comment (Draggeru) [reply code/game/objects/effects/spawners/costume.dm:170]: :(

[the remark this answers is not in the document]
Comment (Draggeru) [reply code/game/objects/effects/spawners/costume.dm]: Educate me on why it must be tape_wizard instead of just tapewizard

**Comment by Draggeru** [comment]
Fuck it, I'm giving people super sticky tape because its objectively funnier.

File: code/game/objects/items/storage/boxes/clothes_boxes.dm [hunk 2]
```diff
@@ -73,6 +83,12 @@
 	new /obj/item/tank/internals/oxygen(src)
 	new /obj/item/gps(src)
 
+/obj/item/storage/box/hero/PopulateContents()
+	new /obj/item/clothing/head/wizard/tape/fake(src)
+	new /obj/item/clothing/suit/wizrobe/tape/fake(src)
+	new /obj/item/staff/tape(src)
+	new /obj/item/stack/sticky_tape(src)
```
Comment (Fikou) [inline code/game/objects/items/storage/boxes/clothes_boxes.dm:90]: what are you doing here

File: code/game/objects/items/storage/boxes/clothes_boxes.dm [hunk 3]
```diff
@@ -27,6 +27,16 @@
...
 
+/obj/item/storage/box/tape_wizard
+	name = "Tape Wizard - Episode 23"
+	desc = "A box containing the costume used by legendary entertainment icon 'Super Tape Wizard'. It got a little stuck on its way out."
+
+/obj/item/storage/box/tape_wizard/PopulateContents()
+	new /obj/item/clothing/head/wizard/tape/fake(src)
+	new /obj/item/clothing/suit/wizrobe/tape/fake(src)
+	new /obj/item/staff/tape(src)
+	new /obj/item/stack/sticky_tape/super(src)
```
Comment (Fikou) [inline code/game/objects/items/storage/boxes/clothes_boxes.dm:38]: no bepis items in a roundstart vendor box

File: code/game/objects/items/storage/boxes/clothes_boxes.dm (hunk 2 above)
Comment (Draggeru) [reply code/game/objects/items/storage/boxes/clothes_boxes.dm:90]: Being stupid and removing the wrong thing, whoops

File: code/game/objects/items/storage/boxes/clothes_boxes.dm (hunk 3 above)
Comment (Draggeru) [reply code/game/objects/items/storage/boxes/clothes_boxes.dm:38]: 1984.

**Comment by Shadowflame909** [comment]
What does the real tape wizards tape do?

File: code/game/objects/effects/spawners/costume.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (CRITAWAKETS) [reply code/game/objects/effects/spawners/costume.dm]: whole codebase works on snake_case when possible, it's just a naming convention

**Comment by Draggeru** [comment]
> What does the real tape wizards tape do?

Nothing, this just adds a costume, places it in the den (and puts the fake version in the autodrobe(and makes a costume spawner). The wiz den has super sticky tape and the autodrobe kit comes with sticky tape

File: code/game/objects/effects/spawners/costume.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mothblocks) [reply code/game/objects/effects/spawners/costume.dm]: ^

**Comment by Loafin34** [comment]
I will enjoy this immensely, IF you can get a tape spell that can tape a person's shoes to the floor, then their feet. that would make this amazingly good.

**Review by Imaginos16 — APPROVED** [review]
Hilarious

File: code/game/objects/items/weaponry.dm [hunk 4]
```diff
@@ -537,6 +537,13 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	inhand_icon_state = "broom"
 	resistance_flags = FLAMMABLE
 
+/obj/item/staff/tape
+	name = "tape staff"
+	desc = "A roll of tape snugly attached to a stick."
+	icon_state = "tapestaff"
+	inhand_icon_state = "tapestaff"
+	resistance_flags = FLAMMABLE
```
Comment (ATH1909) [inline code/game/objects/items/weaponry.dm:545]: give it a harmless embed chance like a real taped object, coward

Comment (Draggeru) [reply code/game/objects/items/weaponry.dm:545]: bet

Comment (Mothblocks) [reply code/game/objects/items/weaponry.dm:545]: ath was asked to stop using code review for balance requests

**Pull request merged**